Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM fluxrm/flux-sched:focal
```

Comment:
What is this new Dockerfile?

Reply:
Oh, I meant to delete that. Similar to the comments, that was for testing.